### PR TITLE
Update Output styleguidist documentation

### PR DESCRIPTION
**Pull request by captainsafia**

### CODE REVIEW

File: packages/outputs/src/components/output.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-The `<Output />` element is a glorified Switch statement for picking what to render an output with.
+The `Output` element takes a single `output` prop and several child components and renders the appropriate components to render each output type.
```
Comment (willingc) [inline packages/outputs/src/components/output.md:1]: ...components needed for each output type.

**Review by rgbkrk — APPROVED** [review]
The main takeaway, at least for hydrogen is that they can completely override how to render a given output. We probably need an example that uses a custom component for Jupyter errors or something like that.

File: packages/outputs/src/components/output.md [hunk 2]
```diff
@@ -14,6 +16,7 @@ const output = Object.freeze({
   <StreamText />
 </Output>;
 ```
+We can also render errors that are returned from the Jupyter kernel we're connected to.
```
Comment (willingc) [inline packages/outputs/src/components/output.md]: Errors returned from the connected Jupyter kernel can be rendered.

**Review by willingc — APPROVED** [review]
Minor suggestions.

**Comment by captainsafia** [comment]
Thanks for the feedback! I made changes.

**Pull request merged**